src/pages/Login/Login.js

```diff
@@ -1,5 +1,85 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
+import { flex } from '../../styles/mixin';
+import styled from 'styled-components';
 
 export default function Login() {
-  return <div>Login</div>;
+  return (
+    <Container>
+      <LogoImg src="images/soomin/logo.png" />
+      <LoginForm>
+        <UserInput type="text" placeholder="이메일" />
+        <UserInput type="text" placeholder="비밀번호" />
+        <LoginButton>로그인</LoginButton>
+        <Signup href="/signup">회원가입</Signup>
+        <Text>SNS계정으로 간편 로그인/회원가입</Text>
+        <section>
+          <Link to="">
+            <LoginImg
+              src="https://cdn-icons-png.flaticon.com/512/3669/3669973.png"
+              alt="kakaoLogin"
+            />
+          </Link>
+        </section>
+      </LoginForm>
+    </Container>
+  );
 }
+const Container = styled.div`
+  ${flex('center', 'center', 'column')}
+  height: 100vh;
+  background-color: ${props => props.theme.background};
+`;
+
+const LogoImg = styled.img`
+  width: 120px;
+`;
+
+const LoginForm = styled.form`
+  ${flex('', 'center', 'column')}
```

Comment:
```suggestion
  ${flex(null, 'center', 'column')}
```
사용하지 않는 속성은 "" 보단 null을 보내주세요.